Microsoft.Toolkit.Uwp.SampleApp/SamplePages/OneDrive Service/OneDriveCode.bind

```diff
@@ -1,131 +1,116 @@
-//  - Add a platform and select mobile application
-//  - Save
-//  In the sample app choose "Microsoft Account" as the authentication provider.
-
-// To use the new converaged application functionality, you must use the OneDriveService.GraphInstance instance.  This new endpoint uses 
-// the Microsoft Authentication Library (MSAL) to call v2 of the Azure AD services, and has no requirement on the OneDrive SDK.
-// The previous authentication models using Active Directory Authentication Library (ADAL) which called v1 of the Azure AD services and had
-// a OneDrive SDK dependency continues to reside under OneDriveService.Instance.  The API surface under OneDriveService.GraphInstance and
-// OneDriveService.Instance remain the same wherever possible, but you will need to switch to the new Microsoft.Toolkit.Services.OneDrive
-// namespace to access the new types (vs the older Microsoft.Toolkit.Uwp.Services.OneDrive types).
-
-// First get the root of your OneDrive
-// By default the service silently connects the current Windows user if Windows is associated with a Microsoft Account and if you choose 
-// the OnlineId authentication provider 
-
-var folder = await OneDriveService.Instance.RootFolderAsync();
-
-// or, if you are calling the new endpoint something like this:
-
-var folder = await OneDriveService.GraphInstance.RootFolderAsync();
-
-// if Windows is not associated with a Microsoft Account, you need to
-// 1) Initialize the service using an authentication provider AccountProviderType.Msa or AccountProviderType.Adal
-
-OneDriveService.Instance.Initialize(appClientId, AccountProviderType.Msa, OneDriveScopes.OfflineAccess | OneDriveScopes.ReadWrite);
-
-// or, if you are calling the new endpoint something like this:
-
-Services.OneDrive.OneDriveService.GraphInstance.Initialize(appClientId, new string[] {"Files.ReadWrite.All"});
-
-// 2) Sign in
+// OneDriveService works for OneDrive, OneDrive for Business, SharePoint document libraries, and Office Groups, to allow your 
+// app the flexibility to read and store content in any of these locations with the same code
+
+// Getting Started
+
+// To use the OneDrive API, you need to have an access token that authenticates your app to a particular set of permissions 
+// for a user. In this section, you'll learn how to:
+
+// 1. Register your application to get a client ID and a client secret.
+// 2. Sign your user in to OneDrive with the specified scopes using the token flow or code flow.
+// 3. Sign the user out (optional).
+
+// Register Your App
+
+// To connect with Microsoft Graph, you'll need a work/school account or a Microsoft account.
+
+// 1. Go to https://apps.dev.microsoft.com/
+// 2. When prompted, sign in with your account credentials.
+// 3. Find My applications and click Add an app.
+// 4. Enter your app's name and click Create application.
+
+// After you've completed these steps, an application ID is created for your app and displayed on your new app's properties page.
```

Comment:
Could you also add a quick note here on selecting Delegated Permissions to access OneDrive (Files.ReadWrite, etc)?

Reply:
will do

Reply:
Done - Added it to docs and bind